enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,271 @@
+
+This approach guarantees have backups across all master nodes which is vital in disaster recovery scenarios.
+
+### User Stories
+
+* As a cluster administrator, I would like OCP cluster backup to be taken without configuration from day one.
+* As a cluster administrator, I would like to schedule recurring OCP cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+
+### API Extensions
+
+No [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) changes are required.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A sidecar container is being deployed within each etcd pod in Openshift cluster in order to take backups according to default configuration using `config.openshift.io/v1alpha1 Backup`.
+
+To distinguish default _no-config_ backups from user configured counterparts, a `config.openshift.io/v1alpha1 Backup` CR's name must be `default`.
+
+Initially, the sidecar container within etcd pods is not running (i.e. disabled). However, creating a `Backup` CR with `name=default` enables the container.
+
+Once enabled, the backups are being taken on each master node according to the provided `schedule` within the CR. Moreover, the backups are being maintained according to the supplied retention policy. 
+
+The default CR's configuration could be overridden by the user according to their preference.
+
+The backups are being stored within a `hostPath` volume. The volume is a directory within each master node's file system.
+
+Before deciding upon this approach, several approaches that have been investigated. See [Alternatives](#Alternatives).
+
+### Open Questions
+
+Currently, the no-config backup sidecar container exits upon error. Since the container is being deployed within the etcd Pod, the whole Pod will never be `Running` as long as the container is failing.
+Once the container fails, it will be automatically restarted according to exponential backoff strategy. This scenario leads to `CrashLoopBackOff` and the etcd Pod will never be `Running`. 
+
+As the no-config backup sidecar is being deployed alongside each etcd member, consequently an error within this container leads to unavailable etcd cluster. Hence, failing the container upon error is not a viable option.
+Also disabling the container from the etcd pod upon error is not recommended, unless requested by the user. As discussed above, enabling and disabling the sidecar container leads to deploying a new version of the static pods.
+This scenario leads to a duration of cluster unavailability. 
```

Comment:
not sure where you got that notion from?

Reply:
so during redeploying staticPods upon _enabling_ or _disabling_ **etcd-backup-container** , the cluster is `api-server` is unavailable  for the revision rollout duration

I have seen this consistently  during testing, pleaser correct me if I am wrong

Reply:
Tying the backup pod to the main etcd pod seems to be particularly fraught with danger, in terms of enabling and disabling, and when the backup has issues. It would make more sense for it to be separate no?